docker/local/Dockerfile

```diff
@@ -19,14 +19,16 @@ RUN echo "deb http://httpredir.debian.org/debian jessie main" > /etc/apt/sources
  && apt-get install -y --no-install-recommends \
 	bash-completion \
 	curl \
+	dkms \
```

Comment:
just to let you know, i have noticed that more packages are missing to use all the functionality of the app inside the container